astro/src/content/blog/fusionauth-re-invent-2023.mdx

```diff
@@ -0,0 +1,59 @@
+---
+publish_date: 2023-11-27
+title: "We'll see you at AWS re:Invent 2023"
+description: FusionAuth is going to AWS re:Invent. Stop by and see us!
+authors: Alex Patterson
+image: /img/blogs/fusionauth-re-invent-2023/fusionauth-template.png
+categories: Events
+tags: events, reinvent, conference
+excerpt_separator: "{/* more */}"
+---
+
+AWS re:Invent is one of the largest technical conferences in the world. In the cloud space, it is the go-to event. This week there will be tens of thousands of attendees in Las Vegas, Nevada. The [FusionAuth team will be among them](/aws-reinvent23), and we'd love to see you there!
+
+{/* more */}
+
+## Why the team is attending
+
+We're going to re:Invent for many reasons.
+
+One is that FusionAuth and our clients often deploy on the AWS public cloud. Some clients use vanilla EC2 instances with RDS, others deploy on solutions like EKS and Fargate (yes, [FusionAuth supports Kubernetes](/docs/v1/tech/installation-guide/kubernetes/fusionauth-deployment)).
+
+AWS re:Invent is an opportunity to meet developers where they are both during talks and in the ever-present hallway track. We look forward to exchanging knowledge and ideas with folks building on the AWS platform.
+
+We're also excited to chat with attendees about authentication. We like to hear from developers about what they are building on top of solutions like FusionAuth, Auth0 and Cognito.
+
+You can migrate from [Auth0](/docs/v1/tech/migration-guide/auth0) to FusionAuth and we're of course happy to help. Migrating away from Cognito isn't as straightforward, but [there are ways](/docs/v1/tech/migration-guide/cognito).
```

Comment:
This transition isn't real smooth.  You could say something like, "In the past, developers have asked us how to migrate from Auth0 and of course we are happy to help."